code/code/misc/damage.cc

```diff
@@ -1114,7 +1116,11 @@ int TBeing::damageEpilog(TBeing* v, spellNumT dmg_type) {
           if (k->desc) {
             k->desc->session.groupKills++;
             k->desc->career.group_kills++;
-            grouplist = k->getName();
+          }
+          // discord group naming
+          if (k->getName() != getName()) {
+            lastname = k->getName();
+            group_members++;
```

Comment:
It doesn't appear like this thing would be in a loop. Thus you'll get group_members = 0 or 1, below, making line 1137 always true.

Reply:
Tho that's probably OK, as we update the counter later on.

Reply:
this and the following conditionals are to ensure the entity with the killing blow is not listed twice